corehq/motech/repeaters/models.py

```diff
@@ -1054,6 +1159,100 @@ def _get_retry_interval(last_checked, now):
     return interval
 
 
+def attempt_forward_now(repeater_stub: RepeaterStub):
+    from corehq.motech.repeaters.tasks import process_repeater_stub
+
+    if not domain_can_forward(repeater_stub.domain):
+        return
+    if not repeater_stub.is_ready:
+        return
+    process_repeater_stub.delay(repeater_stub)
+
+
+def get_payload(repeater: Repeater, repeat_record: SQLRepeatRecord) -> Any:
```

Comment:
The return value of anything that implements `BasePayloadGenerator.get_payload()`. So far, form XML, form JSON, case XML, case JSON, an app ID, a Tastypie-serialized CommCareUser, a Location as JSON.

But that's a very good question, because I think all those are returned a strings.

Reply:
2acaa36

Reply:
Do you think this be a good candidate for just leaving out the type hint for the return value?

Reply:
I think yes. Maybe I'm too biased to answer that question objectively :stuck_out_tongue:

Reply:
On second thought, no. I think it's fine as is since you're using type annotations in this code anyway.